src/utils/anonymizeLink.test.ts

```diff
@@ -0,0 +1,41 @@
+import { anonymizeLink } from './anonymizeLink'
+
+describe('#anonymizeLink', () => {
+  it('does nothing to non-urls', () => {
```

Comment:
@alfetopito @W3stside this is interesting. Probably important for legal reasons. We should try not to leak these